subsys/bluetooth/controller/hci/hci.c

```diff
@@ -791,6 +791,20 @@ static void read_supported_commands(struct net_buf *buf, struct net_buf **evt)
 	rp->commands[39] |= BIT(2);
 #endif /* CONFIG_BT_CTLR_PRIVACY */
 
+#if defined(CONFIG_BT_CTLR_DF)
+#if defined(CONFIG_BT_CTLR_DF_ADV_CTE_TX)
+	rp->commands[39] |= BIT(5) | BIT(6);
+#endif /* CONFIG_BT_CTLR_DF_ADV_CTE_TX */
+#if defined(CONFIG_BT_CTLR_DF_SCAN_CTE_RX)
+	rp->commands[39] |= BIT(7);
+#endif /* CONFIG_BT_CTLR_DF_SCAN_CTE_RX */
+	rp->commands[40] |= BIT(4);
+#endif /* CONFIG_BT_CTLR_DF */
+
+#if defined(CONFIG_BT_CTLR_SYNC_PERIODIC)
+	rp->commands[40] |= BIT(5);
+#endif /* CONFIG_BT_CTLR_SYNC_PERIODIC */
+
```

Comment:
Add comments as to which HCI commands are these, refer to others above. Also, have you used tabs or spaces? the leading alignments dont look like tabs of 8 spaces.

Reply:
Regarding the indentation, it seems like it is tabs, but Github has always been terrible at showing those correctly.

Reply:
Command description added.
Regarding the indentation, there are tabs in the code.